vim/merlin/dune

```diff
@@ -13,9 +13,12 @@
         (ftdetect/merlin.vim as vim/ftdetect/merlin.vim)
         (ftplugin/merlin.vim as vim/ftplugin/merlin.vim)
         (ftplugin/ocaml.vim as vim/ftplugin/ocaml.vim)
+        (ftplugin/ocamlinterface.vim as vim/ftplugin/ocamlinterface.vim)
+        (ftplugin/ocamllex.vim as vim/ftplugin/ocamllex.vim)
```

Comment:
Is it expected that there is no reference to the `https://github.com/ocaml/merlin/pull/1340/files#diff-9be022c3a5969aeabb7003ebd6ab488ed56021a64cbac51be234b491e572d781` here ?

Reply:
Yes, that would need explicit support for the filetype, which last time I checked was missing. Has it been added?

From the commit:
> Menhir filetype is not included as there seem to be no plans to include it and even then it would take a big effort.

Reply:
No, there's no such support, but then why do you need the ``vim/merlin/ftplugin/menhir.vim` file ?
(sorry, not a vim user :-) )

Reply:
Right, I think I understand. The filetype is described by the name of the file in `vim/merlin/ftplugin` and we want to load Merlin in menhir files even if support is not good.

Reply:
That's right, I think it might be better to just remove the file that enables merlin for menhir. It doesn't make any sense to have it, but not install it because merlin doesn't support it.

Reply:
I've kept the reference to the menhir filetype in the commit, but not on the changelog